README.md

```diff
@@ -24,17 +24,22 @@ Paperize helps game designers make better games by leveraging free web tools to
     - Authorized JavaScript Origins: `http://localhost:8080`
     - Authorized Redirect URIs: none
   - from the Library menu, enable Web Fonts Developer API and create an API Key
+  - from the Library menu, enable the Google Drive API and choose the previously created API key
   - from the Library menu, enable the Google Picker API and choose the previously created API Key
   - from the Library menu, enable the Google Sheets API and choose the previously created API Key
   - Open the OAuth Consent Screen menu again, and add the following scopes:
     - Google Drive API  ../auth/drive.appdata
     - Google Drive API  ../auth/drive.file
     - Google Sheets API ../auth/drive.file
-  - `cp example.api_keys.dev.json .api_keys.dev.json`
+  - copy and rename `example.api_keys.dev.json` to `.api_keys.dev.json` (`cp example.api_keys.dev.json .api_keys.dev.json`)
   - modify `.api_keys.dev.json` to contain the keys you just set up
-- `npm install`
-- `npm test`
-- `npm start`
+- **Mac OS**
+  - `npm install`
+  - `npm test`
+  - `npm start`
+- **Windows**
+  - `npm install`
+  - `npm start`
```

Comment:
let's collapse this back to just a single set of instructions, and add a note after `npm test` that says "(optional, may not work on some systems)" or something to that effect

Reply:
Alright! If I commit to this branch do I need to open another pull request? Or will it update this one?